src/Tests/SettingsTest.php

```diff
@@ -106,9 +106,8 @@ public function testAdminSettingsForm() {
     $edit_settings_count = count($edit);
     $expect_settings_count = 21;
 
-    // Post the edits and assert that options are saved.
+    // Post the edits.
     $this->drupalPostForm('admin/config/services/acquia-lift', $edit, t('Save configuration'));
-    $this->assertText(t('The configuration options have been saved.'));
```

Comment:
Delete an assertion that no longer works in D8.6. We don't need this assertion anyway because we have the next ones to tell that "save" was successful.